jQueryI-jQ.js

```diff
@@ -0,0 +1,2 @@
+let creditCard = $("CC");
+console.log(creditCard);
```

Comment:
It looks like you might not be using this file. It's a good idea either to add a comment to the file to note that it isn't being used at the moment, or maybe even remove the file so that it doesn't get confusing later. (But, it's okay if it stays around!)